lib/tasks/projects.rake

```diff
@@ -53,18 +53,19 @@ namespace :projects do
     end
   end
 
-  desc 'Check to see if projects have been removed'
+  desc 'Check to see if projects are still removed/deprecated'
   task check_removed_status: :environment do
     exit if ENV['READ_ONLY'].present?
-    ['npm', 'rubygems', 'packagist', 'cpan', 'clojars', 'cocoapods',
+    # Check if removed/deprecated projects are still deprecated/removed
+    ['pypi', 'npm', 'rubygems', 'packagist', 'cpan', 'clojars', 'cocoapods',
```

Comment:
NB we didn't have pypi in here yet, so we were never re-checking their statuses.